src/TableGateway/Feature/EventFeature.php

```diff
@@ -23,7 +24,7 @@ class EventFeature extends AbstractFeature implements
     /** @var EventManagerInterface */
     protected $eventManager;
 
-    /** @var null */
+    /** @var ?EventInterface */
```

Comment:
`EventFeature\TableGatewayEvent`, no?

Reply:
Yea, I suppose it should be limited to TableGatewayEvent. Im currently adding a custom feature is when I noticed it. But since this is focused on the EventFeature it should be the narrower TableGatewayEvent. Will update here in just a second.

Reply:
Fixed.